.npmrc

```diff
@@ -0,0 +1 @@
+save-exact = true
```

Comment:
I think you added it to fix that lint error, right? I tested here and, unfortunately, it didn't fix :(
To simulate, try to run other `npm i` and after that run the lint script.

It returns the following error: Error: `Cannot find module 'prettier'`